src/components/EventForm/EventForm.module.css

```
@@ -10,14 +10,41 @@
 
 .label {
     display: flex;
+    flex-direction: column;
 }
 
 .label > span {
     width: 200px;
 }
 
-.button {
-    margin-top: 12px;
+.labelFindMap {
+    display: flex;
+    justify-content: space-between;
+}
+
+.btnFindMap {
```

```suggestion
.buttonFindMap {
```

Тут предлагаю не экономить

---

Принято